packages/core/src/components/label/label.tsx

```diff
@@ -18,16 +21,29 @@ export class Label {
     ['danger', 'Gevaar']
   ]);
 
+  @Prop()
+  removable?: boolean;
+
+  @State()
+  hovered?: boolean;
```

Comment:
State is een momentopname: Als de huidige status "hovered" is, is het verleden tijd. `hovering` of `hover` is beter

Reply:
Ik kies voor `hovering` aangezien dat precies de handeling is die je doet